src/libs/ReportUtils.ts

```diff
@@ -4663,6 +4675,78 @@ function shouldDisableThread(reportAction: OnyxEntry<ReportAction>, reportID: st
     );
 }
 
+function getAllAncestorReportActions(report: Report | null | undefined, shouldHideThreadDividerLine: boolean): Ancestor[] {
```

Comment:
This PR caused an issue - https://github.com/Expensify/App/issues/41519

Detailed root cause here - https://github.com/Expensify/App/issues/41519#issuecomment-2093608757